orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -253,6 +253,10 @@ def forward(self, *inputs, **kwargs):
             ):
                 self.time_tracker.start(ORTModuleInitPhase.EndToEnd)
 
+                # need to set device before check embedding sparsity because it uses device
```

Comment:
can we move the check into _export_model in orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py?

Reply:
have moved into